map2loop/config.py

```diff
@@ -217,7 +217,7 @@ def update_from_file(
                             raise e
             else:
                 with open(filename) as url_data:
-                    data = hjson.load(url_data)
+                    data = json.load(url_data)
```

Comment:
Would it be worth catching an error here in case the data can't be opened by json

Reply:
added in https://github.com/Loop3D/map2loop/pull/109/commits/35b90979a195cc5a8a60d4370e85c76160f40286. Also added FileNotFound handle